app/register/components/confirmDialog.tsx

```diff
@@ -35,50 +29,35 @@ export function ConfirmDialog({
     event.currentTarget.innerHTML =
       '<span class="loading loading-ring loading-lg"></span>送信中...'
 
-    const fileName = `${crypto.randomUUID()}.${extention}`
-    const file = await CvtDataUrl2File(imageSrc, fileName)
-    const url = await getServerUrl(fileName)
+    const formElement = document.querySelector('form')
+    const formData = new FormData(formElement as HTMLFormElement)
 
-    await fetch(url, {
-      method: 'PUT',
-      headers: {
-        'Content-Type': fileType,
-        'Content-Disposition': `attachment; filename=${fileName}`,
-      },
-      body: file,
+    fetch('/api/image', {
+      method: 'POST',
+      body: formData,
     })
+      .then((res) => {
+        if (!res.ok) {
+          alert(`画像のアップロードに失敗しました：${res.statusText}`)
+        }
+      })
       .catch((error) => {
-        alert(error)
-        return
+        alert(`画像アップロードエラー：${error}`)
       })
+
+    fetch('/api/user', {
+      method: 'POST',
+      body: formData,
+    })
```

Comment:
It looks like lengthly to post the formData twice.
We may should post the formData to one API,
extract the image file from formData,
then send it to the S3 by POST,
and send user profile to the database when POST image success,
or send error back to client when POST fail.

Reply:
I changed the fetch call time at once.